pagopa/io-app
# fix(Cross): [IOAPPX-326] Fix bug on Android when using `SemiBold` font weight

> [!warning]
> This PR depends on:
> * https://github.com/pagopa/io-app-design-system/pull/293

## Short description
This PR fixes a **critical** UI bug on Android when using the `SemiBold` font weight. The new _**Titillium Sans Pro**_ uses `Semibold` as a font attribute, while the previous _**Titillium Web**_ fonts used `SemiBold`. Android didn't handle this edge case and rendered everything with the fallback system font.
  
## List of changes proposed in this pull request
- Replace `SemiBold` font weight attribute with `Semibold` in all the typographic styles

## How to test
1. Launch the app with an Android physical or virtual device
2. Check **Typography** screen or everything else in the app
